packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -649,7 +652,10 @@ export const GiantTrackTile = ({
         </div>
       </div>
 
-      {isStreamGated && streamConditions ? (
+      {isStreamGated &&
```

Comment:
not sure if this is correct. what if stream gated and still has downloadable assets, should we not still show this? maybe not clear in figma but we should check with design

Reply:
hm i think if stream gated and has downloadable assets, we show the buy button in the stems & download section (top right of the table). but i have not seen a design that explicitly says when to show this section and when to not. i'll check with design.

Reply:
checked with design, seems like what i said above is correct!